hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -914,6 +950,97 @@ private void validateAllColumnStats(
     validate(metadataBasedColStats, fsBasedColStats, partitionPath, "column stats");
   }
 
+  private void validatePartitionStats(
+      HoodieMetadataValidationContext metadataTableBasedContext,
+      Set<String> baseDataFilesForCleaning,
+      List<String> allPartitions) throws Exception {
```

Comment:
let's follow the usual style - if all args fit within 200 chars then no need to break. If there is a need to break, then keep the first arg in the same line as the method, and then each arg on new line indented to the same position as the first arg. I think if you install our checkstyle in the IDE, it should automatically format that way.

Reply:
Addressed